doc/man1/openssl-pkeyutl.pod.in

```diff
@@ -147,6 +155,19 @@ Decrypt the input data using a private key.
 
 Derive a shared secret using the peer key.
 
+=item B<-encap>
+
+Encapsulate the secret using a public key. I<-secret> is required.
+
+=item B<-decap>
+
+Decapsulate the secret using a private key.
+
+=item B<-kemop> I<operation>
+
+KEM operation, specific for the key algorithm. See L<EVP_PKEY_CTX_set_kem_op(3)>
+and algorithm-specific KEM documentation.
```

Comment:
Can anything be said here briefly to explain what this does, or when it is needed?